app/views/candidate_mailer/nudge_unsubmitted_with_incomplete_references/no_references.text.erb

```diff
@@ -1,24 +1,20 @@
 Dear <%= @application_form.first_name %>
 
-You have not requested your teacher training references yet. 
+You have not added your teacher training references yet.
```

Comment:
Do we think it's ok that it'll say this even in the (unlikely) scenario that you’ve added 2+ reference but just not marked the section as complete?

Reply:
'You have not yet completed the reference section of your teacher training application'?

Reply:
I've made a suggestion. It's a bit more technical sounding ("the references section") but more accurate.